text/0039-reduce-module-size.md

```diff
@@ -0,0 +1,533 @@
+action to upgrade
+
+Body: We’ve identified that you are using at least one of the [List of
+Constructs] Constructs. If you are running your aws-cdk commands in an
+environment that does not have access to npm, then you will need to make a
+change to make version 2.x of [npm packages] available in your environment. See
+[#1234](https://github.com/aws/aws-cdk/issues/1234) for details and
+instructions.
+
+| List of Constructs                                                                        | npm packages                                                                                            |
+| ----------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| KubectlLayer                                                                              | @aws-cdk/lambda-layer-kubectl                                                                           |
+| KubectlProvider, HelmChart, KubernetesManifest, KubernetesObjectValue, or KubernetesPatch | @aws-cdk/lambda-layer-kubectl and @aws-cdk/lambda-layer-awscli                                          |
+| AwsCliLayer, BucketDeployment, or EmrContainersStartJobRun                                | @aws-cdk/lambda-layer-awscli                                                                            |
+| NodeProxyAgentLayer, or FargateProfile                                                    | @aws-cdk/lambda-layer-node-proxy-agent                                                                  |
+| aws_eks.Cluster                                                                           | @aws-cdk/lambda-layer-kubectl, @aws-cdk/lambda-layer-awscli, and @aws-cdk/lambda-layer-node-proxy-agent |
+
+---
+
+Ticking the box below indicates that the public API of this RFC has been
+signed-off by the API bar raiser (the `api-approved` label was applied to the
+RFC pull request):
+
+```
+[ ] Signed-off by API Bar Raiser @xxxxx
+```
+
+## Public FAQ
+
+### What are we launching today?
+
+Today, we are launching a smaller version of the AWS CDK v2 framework library,
+`aws-cdk-lib`. And, a change in the AWS CDK v2 CLI to support this smaller
+framework library. Before today’s release, the unpacked size of this package was
+234 MB. This large size prevented many customers from being able to use it, and
+caused performance problems for others. It also prevented the CDK team from
+releasing certain new features. Now, the size of the package is less than 100
+MB. This smaller version also has a mechanism for adding new lambda-layer
+dependencies. Because of this launch, the AWS CDK framework will be able to add
+support for more versions of kubectl, and awscliv2.
+
+### Why should I use this feature?
+
+You should upgrade your dependencies on `aws-cdk-lib` to get the benefits of a
+smaller package size. If you use `aws-cdk-lib` in a lambda function, now you can
```

Comment:
Capitalize "lambda" to "Lambda"?

(This comment applies to other places in the doc as well)